spec/unit/blockscore/collection/member_spec.rb

```diff
@@ -1,7 +1,32 @@
 module BlockScore
   class Collection
     RSpec.describe Member do
-      describe '.save' do
+      describe '#save' do
+        let(:question_set) { QuestionSet.new }
+        subject(:member) { described_class.new(person, question_set) }
+
+        context 'unsaved person' do
+          let(:person) { BlockScore::Person.new(attributes_for(:person)) }
+
+          it 'saves the person as part of member save' do
+            expect(person).to_not be_persisted
+            expect(member.save).to be true
+            expect(person).to be_persisted
+          end
+        end
+
+        context 'previously saved person' do
+          let(:person) { BlockScore::Person.create(attributes_for(:person)) }
+
+          it 'contains the proper fields' do
+            expect(member.save).to be true
+            expect(person).to be_persisted
+            expect(person.attributes.fetch(:question_sets)).to include question_set.id
+            expect(person.attributes.fetch(:question_sets).size).to be 1
```

Comment:
This isn't a blocker for this PR, but instead of these two matches I would probably just write:

```ruby
expect(person.attributes.fetch(:question_sets)).to eql([question_set.id])
```

If the question sets included anything different from what was expected the error message should include the diff of the data. No point in specifying two matches when one can do the same job.